src/JuliusSweetland.OptiKey/Enums/FunctionKeys.cs

```diff
@@ -67,6 +67,7 @@ public enum FunctionKeys
         LeftAlt,
         LeftCtrl,
         LeftShift,
+        RightShift,
```

Comment:
I'm going to ignore this one and remove the commit from the pull request because I don't need it and I don't know enough about modifiers to know whether it's working correctly.